### ISSUE TITLE
Error when using howdoi

### ISSUE THREAD

**Issue by Jamesking56** [gleitz/howdoi#105]
Hey,

I just installed `howdoi` on my Yosemite Macbook using `brew`.

I ran a test (to see if it knew about Composer):

`howdoi install composer`

and the response I got back was this:

````
/Library/Python/2.7/site-packages/requests-2.6.0-py2.7.egg/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
/Library/Python/2.7/site-packages/requests-2.6.0-py2.7.egg/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
curl -s http://getcomposer.org/installer | sudo /Applications/MAMP/bin/php/php5.6.2/bin/php
````

What is with those errors before the answer and how do I get rid of them?

**Comment by colincrossland** [comment]
From : https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning

Certain Python platforms (specifically, versions of Python earlier than 2.7.9) have restrictions in their ssl module that limit the configuration that urllib3 can apply. In particular, this can cause HTTPS requests that would succeed on more featureful platforms to fail, and can cause certain security features to be unavailable.

If you encounter this warning, it is strongly recommended you upgrade to a newer Python version, or that you use pyOpenSSL as described in the OpenSSL / PyOpenSSL section.

If you know what you are doing and would like to disable this and other warnings, please consult the InsecureRequestWarning section for instructions on how to handle the warnings.

tl;dr = update python
